spec/decision_service_spec.rb

```diff
@@ -46,6 +46,19 @@
       expect(Optimizely::Audience).not_to have_received(:user_in_experiment?)
     end
 
+    it 'should return the correct variation ID (using Bucketing ID attrbiute) for a given user for whom a variation has been forced' do
+      user_attributes = {
```

Comment:
Please use PHP values, the problem is that on a PHP-SDK everything is calculated & here we are not sure, what variation should be returned for the given arguments.

Reply:
Let's discuss this